apps/react-app/src/providers/mac.context.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+
+export interface MacContextModel {
+  macData: string;
+  setMacData: (macData:string)=>void
+}
```

Comment:
We could try something like

```ts
export interface UserChoiceModel {
  manfredJsonContent: string;
}

export interface UserChoicecontextModel {
   userChoice : UserChoiceModel;
   setUserChoise : (userChoice: UserChoiceModel) => void;
}
```

Reply:
Let's build a factory for the default _userChoiceModel_ (just to have a safe value when we instantiate the container)

```ts
const generateDefaultUserChoiceModel = () => ({manfredJsonContent: ''});
```